src/scripts/main.js

```diff
@@ -1,3 +1,82 @@
 'use strict';
 
-// write code here
+const container = document.querySelector('.container');
+const table = document.querySelector('table');
+const appendRowBotton = document.querySelector('.append-row');
+const removeRowBotton = document.querySelector('.remove-row');
+const appendColumnBotton = document.querySelector('.append-column');
+const removeColumnBotton = document.querySelector('.remove-column');
+
+container.addEventListener('click', (e) => {
+  const buttonTarget = e.target.closest('button');
+  const tr = table.querySelector('tr');
+
+  const newCell = document.createElement('td');
+  const cellLength = document.querySelectorAll('tr').length;
+
+  if (!buttonTarget) {
+    return;
+  }
+
+  switch (buttonTarget) {
+    case appendRowBotton:
+      if (table.rows.length < 10) {
```

Comment:
try to avoid `magic` numbers such as 2 or 10. 
create variables
const MAX_LENGTH = 10
const MIN_LENGTH = 2